volatility3/framework/plugins/windows/registry/userassist.py

```diff
@@ -335,6 +336,20 @@ def _generator(self):
             )
             yield result
 
+    def generate_timeline(self):
+        self._reg_table_name = intermed.IntermediateSymbolTable.create(
```

Comment:
Is it necessary for this to be loaded before the generator is called?  Feels like it either belongs in the generator or should be parameterized rather than a self variable...

Reply:
Yes, it seemed strange to me too as when I was initially testing the plugin I saw where a sample would get userassist output normally but then my timeline code wasn't reporting anything. I then saw the run function had that setup:

https://github.com/volatilityfoundation/volatility3/blob/8525edd3331c6ccd967c8048d61adf5d2c3e1015/volatility3/framework/plugins/windows/registry/userassist.py#L354

So I copied it over to the generate_timeline and it worked.

I thought of moving that `self._reg_table_name` into the generator, but was nervious of side effects since it also seems to play a role in the registry layer:

https://github.com/search?q=repo%3Avolatilityfoundation%2Fvolatility3%20_reg_table_name&type=code

Although the one set in the registry layer doesn't seem referenced from anywhere.

Reply:
Huh, that is a bit strange.  In this plugin at least I'd like to get it moved to the generator.  In the registry layer, we might reference it just by type name and volatility may figure it out, but that's something we can test another time.  Having it saved but not be referenced is fine, getting it loaded into the context is the important part.

Reply:
Fixed and tested.